tests/unit/test_instance_subcmd.py

```diff
@@ -54,13 +57,14 @@
 # pylint: disable=line-too-long
 INST_ENUM_HELP = """Usage: pywbemcli instance enumerate [COMMAND-OPTIONS] CLASSNAME
 
-  Enumerate instances or names of CLASSNAME.
+  Enumerate CIMInstances or names of CLASSNAME.
 
-  Enumerate instances or instance names (the --name_only option) from the
-  WBEMServer starting either at the top  of the hierarchy (if no CLASSNAME
-  provided) or from the CLASSNAME argument if provided.
+  Enumerate CIMInstances or CIMInstanceNames (the --name_only option) from
```

Comment:
Don't use class names in plural.

Reply:
Changed to say Get CIMInstance or CIMInstanceName (...) objects from ...